### ISSUE TITLE
FTN Lowercase area tags

### ISSUE THREAD

**Issue by apamment** [NuSkooler/enigma-bbs#78]
Hi,

I *think* this is a bug, but if a packet comes in with an area tag written in case that doesn't match the case used in the config file, it throws the message out as it cant find a local area.

I changed getLocalAreaTagByFtnAreaTag to convert to uppercase on the test and it appears to work now.

```
this.getLocalAreaTagByFtnAreaTag = function(ftnAreaTag) {
                return _.findKey(Config.messageNetworks.ftn.areas, areaConf => {
                        return areaConf.tag.toUpperCase() === ftnAreaTag.toUpperCase();
                });
        };
```

**Comment by NuSkooler** [comment]
@apamment : Sounds right! Since it's a small change I'll throw it in now.

**Comment by NuSkooler** [comment]
Resolved @ 359241654b10e856368b81b5306760b86b017c90
